### PR TITLE
Making ubi8 image compatible with s390x

### PR DESCRIPTION
**Pull request by nwang92**

### CODE REVIEW

**Review by bb03 — DISMISSED** [review]

**Comment by nwang92** [comment]
@hendolim since you've got more experience working with UBI8, let me know if there's anything that might have a conflict with your work.

File: base/redhat-8/install.sh [hunk 1]
```diff
@@ -27,24 +28,26 @@ export LANG=en_US.utf8
 
 microdnf -y --nodocs install wget sudo shadow-utils procps
 #install busybox direct from the multiarch since epel isn't availible yet for redhat8
-wget https://busybox.net/downloads/binaries/1.28.1-defconfig-multiarch/busybox-x86_64
-mv busybox-x86_64 /bin/busybox
+wget -O /bin/busybox https://busybox.net/downloads/binaries/1.28.1-defconfig-multiarch/busybox-`arch`
 chmod +x /bin/busybox
-microdnf -y --nodocs install python2 tar
+microdnf -y --nodocs install gcc redhat-rpm-config python2-devel libffi-devel openssl-devel tar
```
Comment (mikedickey) [inline base/redhat-8/install.sh:33]: Is gcc necessary here? How about pip upgrade (versus using latest package)? We need to be careful not to balloon image size. At a minimum, anything no longer needed (like gcc) should be removed before this script completes.

**Review by mikedickey — CHANGES_REQUESTED** [review]

File: base/redhat-8/install.sh (hunk 1 above)
Comment (nwang92) [reply base/redhat-8/install.sh:33]: Oops, I'll address this. I believe I needed the additional packages mostly for the c headers in some of the dependencies used by ansible. I'll try and remove anything that's not necessary

**Comment by nwang92** [comment]
cc @mikedickey 

I removed a few packages that I added in order to get Ansible + the c libs to compile. With these changes, the image size is slighter larger than Debian 10 but it still seems comparable:
```
# docker images
REPOSITORY                                    TAG                 IMAGE ID            CREATED             SIZE
uf-redhat-8                                   latest              30c4c0c15580        4 minutes ago       468MB
base-redhat-8                                 latest              1a75d61c1c34        21 minutes ago      407MB
uf-debian-10                                  latest              92e876d73c91        4 days ago          458MB
base-debian-10                                latest              e6fe1d3e1415        4 days ago          397MB
```

I was able to start the redhat-8 image on the s390x box:
```
# docker run -d -P -e SPLUNK_START_ARGS=--accept-license -e SPLUNK_PASSWORD=helloworld uf-redhat-8:latest
# curl -k -u admin:helloworld https://localhost:32769/services/server/info
<?xml version="1.0" encoding="UTF-8"?>
<!--This is to override browser formatting; see server.conf[httpServer] to disable. . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .-->
<?xml-stylesheet type="text/xml" href="/static/atom.xsl"?>
<feed xmlns="http://www.w3.org/2005/Atom" xmlns:s="http://dev.splunk.com/ns/rest" xmlns:opensearch="http://a9.com/-/spec/opensearch/1.1/">
  <title>server-info</title>
  <id>https://localhost:32769/services/server/info</id>
  <updated>2019-09-24T22:00:30+00:00</updated>
  <generator build="7651b7244cf2" version="7.3.1.1"/>
...
```

**Review by mikedickey — APPROVED** [review]

**Review by bb03 — APPROVED** [review]

**Pull request merged**